### PR TITLE
[Workers] cdn tutorial revision

### PR DESCRIPTION
**Pull request by deadlypants1973**

### CODE REVIEW

File: products/workers/src/content/tutorials/configure-your-cdn/index.md [hunk 1]
```diff
@@ -261,11 +261,11 @@ $ wrangler build
 $ wrangler publish
 ```
 
-After deploying your project, open up your browser to test retrieving your assets. For example, if your Workers project is deployed to the route `myassets.com`, going to the URL `myassets.com/faces/1.jpg` should show an uploaded asset. In addition, you can inspect the request in your browser to confirm that it is being cached. Cloudflare’s CDN will send a `cf-cache-status` header, with `HIT` or `MISS`, to indicate whether the content was cached or not:
+After deploying your project, open your browser to test retrieving your assets. For example, if your Workers project is deployed to the route `myassets.com`, going to the URL `myassets.com/faces/1.jpg` should show an uploaded asset. In addition, you can inspect the request in your browser to confirm that it is being cached. Cloudflare’s CDN will send a `cf-cache-status` header, with `HIT` or `MISS`, to indicate whether the content was cached or not:
 
 ![Result](./media/result.png)
 
-## Related resources
+## Conclusion
```
Comment (deadlypants1973) [inline products/workers/src/content/tutorials/configure-your-cdn/index.md]:
```suggestion
## Related resources
```

File: products/workers/src/content/tutorials/configure-your-cdn/index.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ This tutorial assumes that you have a public bucket on Google Cloud Storage. You
 
 This tutorial makes use of sample images to illustrate serving data through your Cloudflare Workers function. If you have an existing set of images you would like to use, you can upload those to your Google Cloud Storage bucket and use them — if you do not have an existing set of images, this tutorial provides a sample set of profile pictures via [UIFaces.com](http://uifaces.com/), formatted in numeric order (`1.jpg`, `2.jpg`, … `199.jpg`).
 
-To follow along with this tutorial using the data set provided, download the [sample image collection](https://storage.cloud.google.com/workers-docs-configure-your-cdn-tutorial/faces.zip), and upload the zipped folder `faces` to root of your bucket. The directory structure should look like this:
+To follow along with this tutorial using the data set provided, download the [sample image collection](https://storage.cloud.google.com/workers-docs-configure-your-cdn-tutorial/faces.zip) and upload the zipped folder `faces` to root of your bucket. The directory structure should look like this:
```
Comment (abracchi-tw) [inline products/workers/src/content/tutorials/configure-your-cdn/index.md]:
```suggestion
To follow along with this tutorial using the dataset provided, download the [sample image collection](https://storage.cloud.google.com/workers-docs-configure-your-cdn-tutorial/faces.zip) and upload the zipped folder `faces` to the root of your bucket. The directory structure should look like this:
```

File: products/workers/src/content/tutorials/configure-your-cdn/index.md [hunk 3]
```diff
@@ -52,7 +52,7 @@ $ wrangler generate serve-cdn-assets
 $ cd serve-cdn-assets
 ```
 
-By default, Wrangler will use our [`worker-template`](https://github.com/cloudflare/worker-template). Wrangler templates are just Git repositories, so if you want to create your own templates, or use one from our [Template Gallery](/examples), there’s a ton of options to help you get started.
+By default, Wrangler will use our [`worker-template`](https://github.com/cloudflare/worker-template). Wrangler templates are just git repositories, so if you want to create your own templates, or use one from our [Template Gallery](/examples), there is a variety of options to help you get started.
```
Comment (abracchi-tw) [inline products/workers/src/content/tutorials/configure-your-cdn/index.md]:
```suggestion
By default, Wrangler will use our [`worker-template`](https://github.com/cloudflare/worker-template). Wrangler templates are just git repositories, so if you want to create your own templates, or use one from our [Template Gallery](/examples) — there is a variety of options to help you get started.
```

I'd use a stronger punctuation mark there. Also, are we sure about using the word `just` in that context? can we omit it?

File: products/workers/src/content/tutorials/configure-your-cdn/index.md [hunk 4]
```diff
@@ -75,21 +75,21 @@ async function handleRequest(event) {
...
 
 ### Handling requests
 
-Currently, the Workers function receives requests, and returns a simple response with the text `"Hello worker!"`. Begin configuring the function by adding an additional check — requests coming in to the function should only be `GET` requests. If it receives other requests, like `POST`s or `DELETE`s, it should return an error response, with a status code of [`405`](https://httpstatuses.com/405). Using `event.request.method`, the resulting code is below:
+Currently, the Workers function receives requests, and returns a simple response with the text `"Hello worker!"`. Begin configuring the function by adding an additional check — requests coming in to the function should only be `GET` requests. If it receives other requests, like `POST`s or `DELETE`s, it should return an error response, with a status code of [`405`](https://httpstatuses.com/405). Using `event.request.method`, the resulting code is:
```
Comment (abracchi-tw) [inline products/workers/src/content/tutorials/configure-your-cdn/index.md]:
```suggestion
Currently, the Workers function receives requests and returns a simple response with the text `"Hello worker!"`. Begin configuring the function by adding an additional check — requests coming in to the function should only be `GET` requests. If it receives other requests, like `POST`s or `DELETE`s, it should return an error response, with a status code of [`405`](https://httpstatuses.com/405). Using `event.request.method`, the resulting code is:
```

File: products/workers/src/content/tutorials/configure-your-cdn/index.md [hunk 5]
```diff
@@ -105,7 +105,7 @@ async function handleRequest(event) {
 }
 ```
 
-Given that the incoming request to the function is a `GET`, it should be clear that the bulk of your implementation will happen inside of that conditional, replacing the `"Hello worker!"` response. Create a separate function, `serveAsset`, which will house the majority of the implementation for the remainder of the tutorial:
+Given that the incoming request to the function is a `GET`, it should be clear that the majority of your implementation will happen inside of that conditional, replacing the `"Hello worker!"` response. Create a separate function, `serveAsset`, which will house the majority of the implementation for the remainder of the tutorial:
```
Comment (abracchi-tw) [inline products/workers/src/content/tutorials/configure-your-cdn/index.md]:
```suggestion
Given that the incoming request to the function is a `GET`, the majority of your implementation will happen inside of that conditional, replacing the `"Hello worker!"` response. Create a separate function, `serveAsset`, which will house the majority of the implementation for the remainder of the tutorial:
```

File: products/workers/src/content/tutorials/configure-your-cdn/index.md (hunk 2 above)
Comment (marciocloudflare) [inline products/workers/src/content/tutorials/configure-your-cdn/index.md]:
```suggestion
To follow along with this tutorial using the data set provided, download the [sample image collection](https://storage.cloud.google.com/workers-docs-configure-your-cdn-tutorial/faces.zip) and upload the zipped folder `faces` to the root of your bucket. The directory structure should look like this:
```

File: products/workers/src/content/tutorials/configure-your-cdn/index.md [hunk 6]
```diff
@@ -75,21 +75,21 @@ async function handleRequest(event) {
 }
 ```
 
-In your default `index.js` file, you can see that request/response pattern in action. The `handleRequest` constructs a new `Response` with the body text `"Hello worker"`, as well as an explicit `200` status code.
+In your default `index.js` file, you can see that request/response pattern in action. The `handleRequest` constructs a new `Response` with the body text `"Hello worker!"`, as well as an explicit `200` status code.
```
Comment (marciocloudflare) [inline products/workers/src/content/tutorials/configure-your-cdn/index.md]:
```suggestion
In your default `index.js` file, you can observe that request/response pattern in action. The `handleRequest` constructs a new `Response` with the body text `"Hello worker!"`, as well as an explicit `200` status code.
```

Comment (marciocloudflare) [reply products/workers/src/content/tutorials/configure-your-cdn/index.md]: Not sure if `notice` is the better word here, but trying to be more inclusive

**Review by marciocloudflare — APPROVED** [review]

**Review by abracchi-tw — APPROVED** [review]

File: products/workers/src/content/tutorials/configure-your-cdn/index.md (hunk 3 above)
Comment (deadlypants1973) [inline products/workers/src/content/tutorials/configure-your-cdn/index.md]:
```suggestion
By default, Wrangler will use our [`worker-template`](https://github.com/cloudflare/worker-template). Wrangler templates are git repositories, so if you want to create your own templates, or use one from our [Template Gallery](/examples), there is a variety of options to help you get started.
```

**Pull request merged**